chromium_src/chrome/browser/sources.gni

```diff
@@ -23,5 +23,6 @@ if (enable_brave_vpn) {
 }
 
 if (toolkit_views) {
-  brave_chromium_src_chrome_browser_deps += [ "//brave/components/sidebar" ]
+  brave_chromium_src_chrome_browser_deps +=
```

Comment:
why are these here instead of `//brave/browser/sources.gni`? I know this file wasn't created in this PR (it was a long time ago), but what is the criteria for inclusion here?